demo/src/lib/stackblitz/prepareProject.ts

```diff
@@ -0,0 +1,90 @@
+import type {SampleInfo} from '$lib/layout/sample';
+import type {Frameworks} from '$lib/stores';
+import type {Project} from '@stackblitz/sdk';
+import type {StackblitzProcessor} from './utils';
+import {addAsyncFiles, isBootstrapCondition, isDaisyuiCondition} from './utils';
+
+export const createBaseFrameworkProcessors = (): Record<Frameworks, StackblitzProcessor[]> => ({
```

Comment:
Maybe move the `createBaseFrameworkProcessors` and `createFrameworkProcessorsReleased` somewhere else ?
It was surprising to see these functions in the prepareProject file instead of frameworkProcessors

Reply:
As discussed, I renamed the file `unreleasedProcessors`.